libs/utils/commons/src/lib/castcle-name.ts

```diff
@@ -20,110 +20,96 @@
  * Thailand 10160, or visit www.castcle.com if you need additional information
  * or have any questions.
  */
-import {
-  BANNED_NAMES,
-  LENGTH_MAX,
-  LENGTH_MIN,
-  RANDOM_ADJECTIVE,
-  RANDOM_SUBJECTS,
-  RESERVE_NAMES,
-} from './configs/names';
+import { Environment } from '@castcle-api/environments';
+import { CastcleRequest } from './castcle-request';
+import { LENGTH_MAX, LENGTH_MIN } from './configs/names';
 
+type SuggestionName = {
+  swearwords: string[];
+  subjects: string[];
+  adjectives: string[];
+  forbidwords: {
+    high: string[];
+    low: string[];
+  };
+};
+
+let suggestData: SuggestionName;
 export class CastcleName {
```

Comment:
```suggestion
export class CastcleName {
  static toStug(keyword: string): string;
}
```
นอกนั้นย้ายไปไว้ใน repository